INSTALL.md

```diff
@@ -1,5 +1,35 @@
+## Operating system support
+
+Dangerzone can run on a varieties of Operating systems, and have automated tests for a subset of them.
+This section explains what is the support you should expect from us.
+
+You can find general support information in this table, and more details in the following sections.
+
+(Unless specified, the architecture is AMD64)
+
+| Distribution | Supported versions        | Automated tests        | Manual QA         |
+| ------------ | ------------------------- | ---------------------- | ----------------- |
+| Windows      | 3 last releases           | Last release only      | Last version only |
```

Comment:
I'm not sure about the 3 last releases part, given that we currently support Windows 10 and Windows 11. Which is the third one

(also, minor, we refer to "releases", and then the last column refers to a "version")

Reply:
I've updated to "2 last versions" then.

About the version VS releases. I decided to stick to "version" everywhere, as I believe it's clearer this way. See in c0a4f388b36669d0fdbe3726b2f2cc47e7cdca28

Reply:
Hmm, and in the end I went to opposite direction as what we're supporting is more releases than versions, if I read this correctly. See b04135897b7cc1999339254970894f2fb3cae6b